src/util/geojson.js

```diff
@@ -116,10 +150,35 @@ export const getCoordinatesBounds = (coordinates) =>
         ]
     )
 
-const TYPE_NUMBER = 'number'
-const TYPE_STRING = 'string'
+export const CENTROID_FORMAT = {
```

Comment:
Instead of exporting an object with the constants, how about just exporting the constants individually? (less error-prone)
e.g., CENTROID_FORMAT_ARRAY